src/SFA.DAS.Assessor.Functions/Functions/Ofqual/OfqualDataReader.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using CsvHelper;
+using Microsoft.Azure.WebJobs;
+using SFA.DAS.Assessor.Functions.Domain.Entities.Ofqual;
+using SFA.DAS.Assessor.Functions.Domain.OfqualImport.Interfaces;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+
+namespace SFA.DAS.Assessor.Functions.Functions.Ofqual
+{
+    public class OfqualDataReader
+    {
+        private readonly IOfqualDownloadsBlobFileTransferClient _blobFileTransferClient;
+
+        public OfqualDataReader(IOfqualDownloadsBlobFileTransferClient blobFileTransferClient)
+        {
+            _blobFileTransferClient = blobFileTransferClient;
+        }
+
+        [FunctionName(nameof(ReadOrganisationsData))]
+        public async Task<IEnumerable<OfqualOrganisation>> ReadOrganisationsData([ActivityTrigger] string filePath, ILogger logger)
+        {
+            if (!await FileExists(filePath))
+            {
+                logger.LogError("Failed to find Organisations data file at {filePath}.", filePath);
+                return new List<OfqualOrganisation>();
```

Comment:
If this happens then the staging table will be cleared and replaced with nothing - that doesn't sound correct in the case of an error to find the file you have downloaded it shouldn't be clearing the existing data

Reply:
I've changed this to throw a `FileNotFoundException`, which seems appropriate given that this would be an exceptional situation